pom.xml

```diff
@@ -23,13 +32,27 @@
 			<version>1.2.5</version>
 		</dependency>
 
-		<!-- These are the junit dependencies -->
+        <!-- These are the junit dependencies -->
 		<dependency>
```

Comment:
let's get rid of the above comments and keep the pom file clean. I'm guessing junit should have the test scope, but I could be wrong here.

Reply:
Good point. I updated the comments to be more meaningful.